webpack/assets/javascripts/react_app/components/common/dates/ShortDateTime.js

```diff
@@ -34,12 +35,14 @@ ShortDateTime.propTypes = {
   date: PropTypes.any,
   defaultValue: PropTypes.string,
   seconds: PropTypes.bool,
+  showRelativeTimeTooltip: PropTypes.bool,
 };
 
 ShortDateTime.defaultProps = {
   date: null,
   defaultValue: '',
   seconds: false,
+  showRelativeTimeTooltip: false,
```

Comment:
The default value in helper is `true` but default prop value is `false`. Is that intentional?

Reply:
Yes because in the helper it states that it will have relative time so it made sense that it will be there true by default. Do you think I should change it to false and add true values manually in the ruby files?

Reply:
No, I think this is ok, just checking :)